### PR TITLE
Remove embedme

### PR DESCRIPTION
**Pull request by alzimmermsft**
# Description

This PR removes the last usages of embedme from the repository, replaced with `codesnippet-maven-plugin` functionality.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-appservice/README.md [hunk 1]
```diff
@@ -41,14 +41,14 @@ In addition, Azure subscription ID can be configured via environment variable `A
...
+```java readme-sample-authenticate
 AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
 TokenCredential credential = new DefaultAzureCredentialBuilder()
     .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())
     .build();
-AppServiceManager manager = AppServiceManager
-    .authenticate(credential, profile);
+AzureResourceManager azure = AzureResourceManager
+    .authenticate(credential, profile)
+    .withDefaultSubscription();
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-appservice/README.md:51]: It could be a issue of picking the wrong sample tag. It should use tag from
resourcemanager-appservice/src/samples/java/com/azure/resourcemanager/appservice/ReadmeSamples.java

I could fix it after your PR.

**Review by weidongxu-microsoft — APPROVED** [review]
Thanks for the help.

There might be a small issue in comment, but we can fix it after your PR.

File: sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java [hunk 2]
```diff
@@ -34,12 +34,9 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 /**
- * WARNING: MODIFYING THIS FILE WILL REQUIRE CORRESPONDING UPDATES TO README.md FILE. LINE NUMBERS ARE USED TO EXTRACT
- * APPROPRIATE CODE SEGMENTS FROM THIS FILE. ADD NEW CODE AT THE BOTTOM TO AVOID CHANGING LINE NUMBERS OF EXISTING CODE
- * SAMPLES.
- * <p>
  * Class containing code snippets that will be injected to README.md.
  */
+@SuppressWarnings("unused")
```
Comment (srnagar) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java:39]: Why is this needed?

File: sdk/resourcemanager/azure-resourcemanager/pom.xml [hunk 3]
```diff
@@ -270,6 +270,24 @@
...
+              <root>../README.md</root>
+              <glob>**/README.md</glob>
+            </additionalReadme>
+            <additionalReadme>
+              <root>../docs</root>
+              <glob>**/*.md</glob>
+            </additionalReadme>
+          </additionalReadmes>
+        </configuration>
+      </plugin>
```
Comment (srnagar) [inline sdk/resourcemanager/azure-resourcemanager/pom.xml:290]: Can we move this to the parent pom and not have to configure it here?

File: sdk/resourcemanager/azure-resourcemanager/src/samples/java/com/azure/resourcemanager/ReadmeSamples.java [hunk 4]
```diff
@@ -41,57 +41,53 @@
...
 
     private final String rgName = "rg-test";
 
-    // extra empty lines to compensate import lines
-
-
-
-
-
-    // THIS LINE MUST BE AT LINE NO. 60
```
Comment (srnagar) [inline sdk/resourcemanager/azure-resourcemanager/src/samples/java/com/azure/resourcemanager/ReadmeSamples.java:60]: :laughing:

**Review by srnagar — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager/pom.xml (hunk 3 above)
Comment (alzimmermsft) [reply sdk/resourcemanager/azure-resourcemanager/pom.xml:290]: This is specific to resourcemanager, same goes for configurations like this in other POMs, so it won't be able to be moved into the parent POM.

File: sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java (hunk 2 above)
Comment (alzimmermsft) [reply sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java:39]: It's not, I was just struggling with IntelliJ flagging APIs to be removed for being unused

File: sdk/resourcemanager/azure-resourcemanager-appservice/README.md (hunk 1 above)
Comment (alzimmermsft) [reply sdk/resourcemanager/azure-resourcemanager-appservice/README.md]: I've fixed this, turns out a README root and glob was updating codesnippets in other packages.
